persistence/src/main/java/de/cyface/persistence/model/GeoLocationV6.java

```diff
@@ -0,0 +1,320 @@
+        this.speed = speed;
+        this.accuracy = accuracy;
+        this.verticalAccuracy = verticalAccuracy;
+    }
+
+    /**
+     * @return The captured latitude of this data point in decimal coordinates as a value between -90.0 (south pole)
+     *         and 90.0 (north pole).
+     */
+    public double getLat() {
+        return lat;
+    }
+
+    /**
+     * @return The captured longitude of this data point in decimal coordinates as a value between -180.0 and
+     *         180.0.
+     */
+    public double getLon() {
+        return lon;
+    }
+
+    /**
+     * @return The captured altitude of this data point in meters above WGS 84 of available.
+     */
+    public Double getAltitude() {
+        return altitude;
+    }
+
+    /**
+     * @return The current speed of the measuring device according to its location sensor in meters per second.
+     */
+    public double getSpeed() {
+        return speed;
+    }
+
+    /**
+     * @return The current accuracy of the measuring device in centimeters.
+     */
+    public double getAccuracy() {
+        return accuracy;
+    }
+
+    /**
+     * @return The current vertical accuracy of the measuring device in meters if available.
+     */
+    public Double getVerticalAccuracy() {
+        return verticalAccuracy;
+    }
+
+    /**
+     * @return {@code True} if this location is considered "clean" by the provided {@link LocationCleaningStrategy}.
+     */
+    @SuppressWarnings("unused") // Part of the API
+    public Boolean isValid() {
+        return isValid;
+    }
+
+    /**
+     * @return The device-unique id of the measurement this data point belongs to.
+     */
+    public long getMeasurementId() {
+        return measurementId;
+    }
+
+    /**
+     * @param valid {@code True} if this location is considered "clean" by the provided
+     *            {@link LocationCleaningStrategy}.
+     */
+    public void setValid(boolean valid) {
+        isValid = valid;
+    }
+
+    /**
+     * @param measurementId The device-unique id of the measurement this data point belongs to.
+     */
+    public void setMeasurementId(long measurementId) {
+        this.measurementId = measurementId;
+    }
+
+    /*
+     * MARK: Parcelable Interface
+     */
+
+    /**
+     * Constructor as required by {@code Parcelable} implementation.
+     *
+     * @param in A {@code Parcel} that is a serialized version of a data point.
+     */
+    @Ignore // Parcelable requires this constructor, make {@code Room} ignore this constructor.
+    protected GeoLocationV6(final @NonNull Parcel in) {
+        super(in);
+        lat = in.readDouble();
+        lon = in.readDouble();
+        altitude = in.readDouble();
+        speed = in.readDouble();
+        accuracy = in.readDouble();
+        verticalAccuracy = in.readDouble();
+        isValid = in.readByte() != 0;
+        measurementId = in.readLong(); // FIXME: Added, do we need this? GeoLocation has no such field
```

Comment:
Probably should include the id in the parcel, as that would be the expected behaviour. It might be always `null` for now, but that might change and it will be a hard to find bug, why the identifier is missing from the deserialized instances.